services/index.js

```diff
@@ -0,0 +1,38 @@
+const axios = require('axios');
+const tmdb = require('./tmdb');
+const omdb = require('./omdb');
+
+module.exports.updateConfigurations = async () => {
+  const configResponse = await tmdb.requestConfigurations();
+  return configResponse.data;
+};
+
+module.exports.getMovies = async page => {
+  const response = await tmdb.requestDiscoverMovies(page);
+  return response.data;
+};
+
+module.exports.getMoviesData = async movies => {
+  const tmdbRequests = movies.map(movie => tmdb.requestMovieDetails(movie.id));
+  const tmdbResponses = await axios.all(tmdbRequests);
+
+  let movieData = tmdbResponses.map(response => response.data);
```

Comment:
You never reassign, so this should be a `const` instead of `let`

```suggestion
  const movieData = tmdbResponses.map(response => response.data);
```

Reply:
I change the elements inside the movieData array, so I want to be explicit that this object is being mutated in some way.